reporters/reporter-default/src/test/java/TimelineChartsTest.java

```diff
@@ -47,12 +53,21 @@ public void test() {
 
       Cluster cluster = new Cluster();
       cluster.addGroup("default", 2);
-      TimelineDocument doc = new TimelineDocument(new TimelineDocument.Configuration(), "test", "testconfig", "Test Config", Arrays.asList(t0, t1), cluster);
+      TimelineDocument timelineDocument = new TimelineDocument(new TimelineDocument.Configuration(), "test", "testconfig", "Test Config", Arrays.asList(t0, t1), cluster);
+
+      freemarker.template.Configuration cfg = HtmlReporter.initConfig();
       try {
-         doc.open();
-         doc.writeTimelines();
-         doc.close();
-      } catch (IOException e) {
+         timelineDocument.createReportDirectory();
+         timelineDocument.generateTimelineCharts();
+
+         Template reportTemplate = cfg.getTemplate("timelineReport.ftl");
+         PrintWriter printWriter = new PrintWriter(timelineDocument.getDirectory() + File.separator + timelineDocument.getFileName());
+
+         Map root = new HashMap();
+         root.put("timelineDocument", timelineDocument);
+         reportTemplate.process(root, printWriter);
+         printWriter.close();
```

Comment:
One more candidate for https://github.com/radargun/radargun/pull/248/files#r35864832.